modules/ballerina-test/src/test/java/org/ballerinalang/test/statements/whilestatement/WhileStmtTest.java

```diff
@@ -15,15 +15,13 @@
 * specific language governing permissions and limitations
 * under the License.
 */
-package org.ballerinalang.model.statements;
+package org.ballerinalang.test.statements.whilestatement;
 
```

Comment:
Shall we move this to package package org.ballerinalang.test.statements.while. Change the resource bal path accordingly.

Reply:
We cannot use `while` in a package path since it is a keyword in java.